dataactvalidator/README.md

```diff
@@ -33,26 +33,33 @@ Example output: None
-
-Each record is checked against the set of rules for the fields present in the file, with all errors and warnings recorded.  With the exception of records that fail a data type check, each record goes into a staging table in the validation database. Data that fails any validation rule is written to either an error report or warning report, and a running sum is kept of occurrences for each rule, which are written to the error database after all rows have been checked.  Finally, the job is marked as finished in the job tracker, the file is marked completed in the error database.
-
-## Available Validations
-The available rule types are as follows:
-
-* Type checks - verifies that data fits the specified type
-* Equal / Not equal - checks that value is equal to a specified value, or not equal
-* Less than / Greater than - compares value against a specified reference point
-* Length - checks that value is no longer than specified length, this is treated as a warning
-* Set membership - checks that value is one of an allowed set of values
-* Minimum length - checks that the field has at least the number of characters specified
-* Conditional requirement - checks that the field is populated if another specified rule passes
-* Sum - Checks that a field is equal to a sum of other fields
-* Sum to value - Checks that a set of fields sums to a specified value
-* Require one of - Checks that at least one of a set of fields is populated
-* TAS - Check that a TAS number is present in CARS file
-* Field match - Checks that a set of field values is present in another file
-* Conditional rule - Checks first rule if and only if second rule passes
+The file location on S3 is specified in the job tracker, and the validator streams the file record by record from S3.
+
+The validation process for each submitted group of files happens in four steps:
+
+1. Each individual file is checked for correct formatting and a correct header row.
+2. Basic schema checks are performed on each row of each individual file:
+    * are required fields present?
+    * is the data type of each field correct?
+    * is the field length correct? (warning)
+3. The complex validation rules are performed on each individual file.
+4. Once the individual files have passed the previous validation steps, the validator runs a series of "cross-file" checks to ensure that data is consistent between the files.
+
+Finally, the job is marked as finished in the job table, and the file is marked completed in the error metadata table.
+
+Some of the validation rules are defined as in data standard as `errors` and some as `warnings`. Any data that fails a rule marked as `error` will result in a failed validation status, and users will have to fix the problematic data and re-submit.
```

Comment:
I'm confused here by the phrase "as in data standard".  I'm not sure what this means, so I'm guessing some users may also be confused.  Would it be possible to clarify this statement?  Alternatively, the sentence would seem to make sense with those four words removed.